FreeRTOS/FreeRTOS
# Blinky Demo for Cortex M3 MPS2 on Qemu

Blinky Demo for Cortex M3 MPS2 on Qemu

Description
-----------
Implement the Blinky demo that runs on qemu for an emulated Cortex M3 MPS2 board

Test Steps
-----------
Ran Manually on Quemu:
```
parallels@parallels-Parallels-Virtual-Platform:~$ qemu-system-arm -machine mps2-an385  -monitor null -semihosting --semihosting-config enable=on,target=native -kernel /media/psf/Home/mps2_demo  -serial stdio -nographic
blinking
blinking
blinking
blinking
blinking
blinking
blinking
blinking
qemu-system-arm: terminating on signal 2
parallels@parallels-Parallels-Virtual-Platform:~$ 
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
